src/index.html

```diff
@@ -1,12 +1,430 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
```

Comment:
Don't add classes to `html` tag unless it is completely necessary.
Work with `body` tag to make global block for page layout.